ui/search.html

```diff
@@ -23,79 +23,111 @@
+                                        {/if}
+                                    </div>
+                                --></div>
+
+                                <div id="search-template"><!--
+                                    {for result in results}
+                                        ${listItem(result)}
+                                    {/for}
+                                --></div>
+                            </div>
                         </div>
+
+                        <div id="search-lhnavigation-template"><!--
+                            {var tenant = oae.data.me.tenant}
+                            <ul id="search-refine-type" class="nav nav-list">
+                                <li class="nav-header">__MSG__REFINE_YOUR_SEARCH_COLON__</li>
+                                <li class="search-refine-type-user">
+                                    <label class="checkbox">
+                                        <input type="checkbox" data-type="user" />
+                                        <span>__MSG__PEOPLE__</span>
+                                    </label>
+                                </li>
+                                <li class="search-refine-type-group">
+                                    <label class="checkbox">
+                                        <input type="checkbox" data-type="group" />
+                                        <span>__MSG__GROUPS__</span>
+                                    </label>
+                                </li>
+                                <li class="search-refine-type-content">
+                                    <label class="checkbox">
+                                        <input type="checkbox" data-type="content" />
+                                        <span>__MSG__CONTENT__</span>
+                                    </label>
+                                </li>
+                                <li class="search-refine-type-folder">
+                                    <label class="checkbox">
+                                        <input type="checkbox" data-type="folder" />
+                                        <span>__MSG__FOLDERS__</span>
+                                    </label>
+                                </li>
+                                <li class="search-refine-type-discussion">
+                                    <label class="checkbox">
+                                        <input type="checkbox" data-type="discussion" />
+                                        <span>__MSG__DISCUSSIONS__</span>
+                                    </label>
+                                </li>
+                            </ul>
+                            <ul id="search-refine-tenant" class="nav nav-list">
+                                <li class="nav-header">__MSG__SEARCH_ONLY_WITHIN_COLON__</li>
+                                <li class="search-refine-tenant-${tenant.alias|encodeForHTMLAttribute}">
```

Comment:
We probably don't need the `search-refine-tenant-${tenant.alias|encodeForHTMLAttribute}"` class here?